actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -458,6 +460,15 @@ def make_set_cookie_header(header)
         def write_cookie?(cookie)
           request.ssl? || !cookie[:secure] || always_write_cookie
         end
+
+        def set_expires_attribute(options)
+          expires_at, expires_in = options[:expires_at], options[:expires_in]
```

Comment:
These should probably `delete`.

Might be worth asserting that only one of `:expires`, `:expires_at`, `:expires_in` is set?

I do think that, even if we add these friendlier aliases, `:expires` should remain fully supported. In fact, alternative API suggestion: only use `:expires`, but make both `expires: DateTime` and `expires: Duration` DWIM.